libs/create-langchain-integration/template/package.json

```diff
@@ -33,7 +33,7 @@
   "author": "LangChain",
   "license": "MIT",
   "dependencies": {
-    "@langchain/core": "~0.1.2"
+    "@langchain/core": "~0.1.3"
```

Comment:
Thoughts on changing this to `latest` @jacoblee93 ?

Reply:
How about `0.1`? Would that resolve to latest minor version?